### PR TITLE
Hide internal module names used by ChapelStandard

### PR DESCRIPTION
**Pull request by bradcray**
This is a follow-up to #19306 which took pains to keep internal module symbols
visible to code by default via its use of `ChapelStandard`.  We don't generally
want to make these internal module names known/knowable to users, so this
PR hides them again and updates the one place I found that had been relying
on them being auto-available.

### CODE REVIEW

**Comment by bradcray** [comment]
@mppf: See what you think.
@e-kayrakli : Tagging you since the one place that was relying on the `Bytes`/`String` module names was developed (or last touched?) by you.

**Review by e-kayrakli — APPROVED** [review]

File: modules/internal/ChapelStandard.chpl [hunk 1]
```diff
@@ -26,46 +26,46 @@ module ChapelStandard {
   // Internal, but uses standard/CommDiagnostics
 
   // Internal modules.
```
Comment (mppf) [inline modules/internal/ChapelStandard.chpl:28]: Let's put a `note: these public uses do not make the module name itself available` here

**Review by mppf — APPROVED** [review]

**Comment by mppf** [comment]
I think an interesting next step would be to make these internal modules all into submodules of ChapelStandard (so you would e.g. `import ChapelStandard.Bytes` in the one code change here). That would avoid naming conflicts with any user-level modules / mason packages that happen to want the same name.

**Comment by bradcray** [comment]
Proposed the idea in https://github.com/chapel-lang/chapel/issues/19793.

**Pull request merged**